app/views/layouts/_header.html.erb

```diff
@@ -9,7 +9,15 @@
         <span class="icon-bar"></span>
         <span class="icon-bar"></span>
       </button>
-      <%= link_to(image_tag("3sixd.png", :class => "navbar-brand", :alt => "Company logo"), root_path) %>
+      <% if Company.last.logo.present? %>
```

Comment:
@leonardk1 It is not a permanent thing, you were supposed to ask first why did i do this, It is supposed to be `current_user.company` but we don't have the current_user implemented, until it is implemented all of this will go away